catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -244,6 +245,19 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
         responseProps.put(SUGGESTION_RESULT_KEY, (Serializable) suggestionResults);
       }
 
+      if (solrResponse.getSpellCheckResponse() != null
```

Comment:
are there unit tests for this?

Reply:
Should've put [WIP] in my PR. Working on them now.